mithril-fake-aggregator/src/shared_state.rs

```diff
@@ -0,0 +1,1353 @@
+use std::{collections::BTreeMap, path::Path, sync::Arc};
+
+use anyhow::{anyhow, Context};
+use serde_json::Value;
+use tokio::sync::RwLock;
+use tracing::{debug, trace};
+
+use crate::StdResult;
+
+pub struct AppState {
+    epoch_settings: String,
+    certificate_list: String,
+    certificates: BTreeMap<String, String>,
```

Comment:
Why did you use `BTreeMap` here (as there is no apparent need to have the certificate hashes ordered)?
Same comment for the other ones below.

Reply:
That's true but it costs us nothing so I chose to ensure lists will be immutable.